<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerVideoIOModule</class>
 <widget class="qSlicerWidget" name="qSlicerVideoIOModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="1">
      <widget class="qMRMLNodeComboBox" name="VideoNodeSelector">
       <property name="nodeTypes">
        <stringlist>
         <string>vtkMRMLSequenceNode</string>
        </stringlist>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="nodeSelectorLabel">
       <property name="text">
        <string>Current video sequence:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="CodecSelector"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="encodingLabel">
       <property name="text">
        <string>Encoding type:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="parameterLabel">
       <property name="text">
        <string>Encoding parameters:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QTableWidget" name="EncodingParameterTable">
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Value</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Description</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="EncodeButton">
         <property name="text">
          <string>Encode</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qSlicerVideoIOModule.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>qSlicerVideoIOModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>VideoNodeSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
